arches/app/models/card.py

```diff
@@ -202,6 +204,14 @@ def save(self):
 
         """
         with transaction.atomic():
+            if self.graph.publication is not None:
```

Comment:
To make this a little lazier
```suggestion
            if self.graph.publication_id:
```